README.md

```diff
@@ -1,41 +1,57 @@
-# Microsoft Container Registry (MCR)
+# Microsoft Artifact Registry (MAR)
 
-Microsoft Container Registry (MCR) is the primary Registry for all Microsoft Published docker images that offers a reliable and trustworthy delivery of container images with a syndicated catalog, while maintaining the quality that customers expect from a  Microsoft product offering. 
+Microsoft Artifact Registry (also known as Microsoft Container Registry or MCR) is the primary Registry for all Microsoft Published artifacts (such as container images) that offers a reliable and trustworthy delivery of artifacts with a syndicated catalog, while maintaining the quality that customers expect from a  Microsoft product offering. 
 
-For more background on MCR:
+For more background on MAR:
 
 - [Microsoft syndicates container catalog (mcr.microsoft.com)](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/)
 - [Improved discovery experience for Microsoft containers on Docker Hub
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
 
-The discovery experience for MCR is provided through [dockerhub](https://hub.docker.com/publishers/microsoftowner).
+The discovery experience for MAR is provided through [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).
 
-To query the list of repositories within mcr: https://mcr.microsoft.com/v2/_catalog
+To query the list of repositories within MAR: https://mcr.microsoft.com/v2/_catalog

```

Comment:
Should we route users to the portal instead?

Reply:
That's what the line 13 hopes to provide. Providing users the opportunity to browse

By keeping this line, this allows customers to query the repos without the portal. Thoughts?